src/components/Session/Session.tsx

```diff
@@ -42,7 +43,9 @@ export const updateSessionCache = <T extends LoginMutation | RegisterMutation>(
 };
 
 export const useBetaFeatures = () =>
-  useSession().powers?.includes('BetaFeatures');
+  Object.entries(useSession().betaFeatures || {}).flatMap(([key, value]) =>
+    value ? key : []
+  );
```

Comment:
I'd prefer all this wrapped in a `new Set(...)` as that is a better shape to output.

The return type should be specified as
```ts
(): ReadonlySet<keyof BetaFeaturesFragment>
```